meta-mel/conf/distro/mel.conf

```diff
@@ -175,11 +175,6 @@ LINUX_VERSION_TAG ??= "v${@'.'.join('${LINUX_VERSION}'.split('.')[:2])}"
 LINUX_VERSION ??= ""
 BB_GIT_SHALLOW_REVS:pn-linux-mel ??= "${@'${LINUX_VERSION_TAG}' if '${LINUX_VERSION_TAG}' and '${LINUX_VERSION}' else ''}"
 
-# Export path variables into the devshell for convenience
-OE_TOPDIR = "${TOPDIR}"
-OE_WORKDIR = "${WORKDIR}"
-OE_TERMINAL_EXPORTS += "OE_TOPDIR OE_WORKDIR COREBASE"
```

Comment:
Shouldn't we keep it if it's pending upstream?

Reply:
Yeah, I'll move that one into a separate RFC/WIP PR until it gets merged.

Reply:
Rebased and moved this to a different branch.